src/app/product.service.ts

```diff
@@ -33,4 +33,8 @@ export class ProductService {
   update(productId, product): void {
     this.database.object('/products/' + productId).update(product);
   }
+
+  delete(productId) {
```

Comment:
You can set type to void and remove the return

Reply:
Done